fix(docs): batcher and explorer urls (#1688)
yetanotherco/aligned_layer: docs/3_guides/0_submitting_proofs.md

If you run the examples below, make sure you are in Aligned's repository root.

> You can check your submitted proofs on our explorer at https://explorer.alignedlayer.com (for Holesky: https://holesky.explorer.alignedlayer.com).

Reviewer: ```suggestion
You can check your submitted proofs on [Mainnet Explorer](https://explorer.alignedlayer.com) and [Holesky Explorer](https://holesky.explorer.alignedlayer.com).
```

Reviewer 2: +1

```bash
aligned deposit-to-batcher \

> --rpc_url https://ethereum-rpc.publicnode.com \

Reviewer: Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error

```bash
aligned get-user-balance \

> --rpc_url https://ethereum-rpc.publicnode.com \

Reviewer: Use holesky on the predefined examples.

These commands allow the usage of the following flags:

- `--rpc_url` to specify the rpc url to be used.
- `--network` to specify the network to be used. Can be `devnet`, `holesky` or `mainnet`.
- `--user_addr` the address of the user that funded the Batcher.

## 3. Submit your proof to the batcher

This guide will focus on how to submit proofs using the Aligned CLI. To integrate the proof submission process into your application, check the [First Aligned Application tutorial](../3_guides/2_build_your_first_aligned_application.md) where we explain how to generate and submit a proof using the Aligned SDK.

Proof submission is done via the `submit` command of the Aligned CLI. The arguments for the submit command are:

* `proving_system`: The proving system corresponding to the proof you want to submit.
* `proof`: The path of the proof associated to the computation to be verified.
* `vm_program`: When the proving system involves the execution of a program in a zkVM, this argument is associated with the compiled program or some other identifier of the program.
* `pub_input`: The path to the file with the public input associated with the proof.

> * `batcher_url`: The batcher websocket URL. Can be:
>   * mainnet: `wss://mainnet.batcher.alignedlayer.com`
>   * holesky: `wss://batcher.alignedlayer.com`

Reviewer 2: ```suggestion
* `batcher_url`: The batcher websocket URL. Can be:
  * mainnet: `wss://mainnet.batcher.alignedlayer.com`
  * holesky: `wss://batcher.alignedlayer.com`
  * devnet: `ws://localhost:8080`
```

--proving_system SP1 \
--proof <proof_file> \
--vm_program <vm_program_file> \

> --batcher_url wss://mainnet.batcher.alignedlayer.com \

Reviewer: Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error

--proving_system SP1 \
--proof ./scripts/test_files/sp1/sp1_fibonacci.proof \
--vm_program ./scripts/test_files/sp1/sp1_fibonacci.elf \

> --batcher_url wss://mainnet.batcher.alignedlayer.com \

Reviewer: Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error

--proof <proof_file> \
--vm_program <vm_program_file> \
--pub_input <pub_input_file> \

> --batcher_url wss://mainnet.batcher.alignedlayer.com \

Reviewer: Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error

--proof ./scripts/test_files/risc_zero/fibonacci_proof_generator/risc_zero_fibonacci.proof \
--vm_program ./scripts/test_files/risc_zero/fibonacci_proof_generator/fibonacci_id.bin \
--public_input ./scripts/test_files/risc_zero/fibonacci_proof_generator/risc_zero_fibonacci.pub \

> --batcher_url wss://mainnet.batcher.alignedlayer.com \

Reviewer: Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error

--proof <proof_file> \
--public_input <public_input_file> \
--vk <verification_key_file> \

> --batcher_url wss://mainnet.batcher.alignedlayer.com \

Reviewer: Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error

--proof ./scripts/test_files/gnark_plonk_bn254_script/plonk.proof \
--public_input ./scripts/test_files/gnark_plonk_bn254_script/plonk_pub_input.pub \
--vk ./scripts/test_files/gnark_plonk_bn254_script/plonk.vk \

> --batcher_url wss://mainnet.batcher.alignedlayer.com \

Reviewer: Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error
